### PR TITLE
feat(route): add 太原市人力资源和社会保障局

### PR DESCRIPTION
**Pull request by 2PoL**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/gov/taiyuan/rsj/gzdt
/gov/taiyuan/rsj/tyxw
/gov/taiyuan/rsj/gggs
/gov/taiyuan/rsj/xqdt
/gov/taiyuan/rsj/gndt
/gov/taiyuan/rsj/tpxw
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

### CODE REVIEW

File: lib/v2/gov/taiyuan/rsj.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+module.exports = async (ctx) => {
+    const categoryID = ctx.params.caty;
+    const page = ctx.params.page ?? '1';
+
+    const pageParam = parseInt(page) > 1 ? `_${page}` : '';
+    const pagePath = `/zfxxgk/${categoryID}/index${pageParam}.shtml`;
+
+    const currentURL = new URL(pagePath, rootURL);
+    const response = await got({ method: 'get', url: currentURL });
```
Comment (TonyRL) [inline lib/v2/gov/taiyuan/rsj.js:15]:
```suggestion
    const response = await got(currentURL.href);
```

File: lib/v2/gov/taiyuan/rsj.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    const $ = cheerio.load(response.data, { decodeEntities: false });
+    const title = $('.tit').find('a:eq(2)').text();
+    const list = $('.RightSide_con ul li')
+        .map((_, item) => {
+            const tag = $(item).find('a');
+            const tag2 = $(item).find('span.fr');
+            return {
+                title: tag.text(),
+                link: tag.attr('href'),
+                pubDate: timezone(tag2.text(), +8),
```
Comment (TonyRL) [inline lib/v2/gov/taiyuan/rsj.js:30]:
```suggestion
            const link = $(item).find('a');
            const date = $(item).find('span.fr');
            return {
                title: link.attr('title'),
                link: link.attr('href'),
                pubDate: timezone(parseDate(date.text(), 'YYYY-MM-DD'), +8),
```

File: lib/v2/gov/maintainer.js [hunk 3]
```diff
@@ -46,4 +46,5 @@ module.exports = {
     '/huizhou/zwgk/:category?': ['Fatpandac'],
     '/sichuan/deyang/govpulicinfo/:countyName': ['zytomorrow'],
     '/xuzhou/hrss/:category?': ['nczitzk'],
+    '/taiyuan/rsj/:caty/:page?': ['2PoL'],
```
Comment (TonyRL) [inline lib/v2/gov/maintainer.js:49]: Please add new route entries in alphabetical order

File: lib/v2/gov/radar.js [hunk 4]
```diff
@@ -702,4 +702,15 @@ module.exports = {
             },
         ],
     },
+    'rsj.taiyuan.gov.cn': {
```
Comment (TonyRL) [inline lib/v2/gov/radar.js:705]: Please add new radar entries in alphabetical order

File: lib/v2/gov/router.js [hunk 5]
```diff
@@ -38,4 +38,5 @@ module.exports = function (router) {
     router.get('/huizhou/zwgk/:category?', require('./huizhou/zwgk/index'));
     router.get('/sichuan/deyang/govpulicinfo/:countyName', require('./sichuan/deyang/govpulicinfo'));
     router.get('/xuzhou/hrss/:category?', require('./xuzhou/hrss'));
+    router.get('/taiyuan/rsj/:caty/:page?', require('./taiyuan/rsj'));
```
Comment (TonyRL) [inline lib/v2/gov/router.js:41]: Please add new route entries in alphabetical order

**Comment by 2PoL** [comment]
I made all the entries in these files alphabetical, not quite sure if that's right.

**Pull request merged**